src/bio.c

```diff
@@ -216,6 +219,11 @@ void *bioProcessBackgroundJobs(void *arg) {
             if (job->fd_args.need_fsync) {
                 redis_fsync(job->fd_args.fd);
             }
+            if (job->fd_args.need_reclaim_cache) {
+                if (reclaimFilePageCache(job->fd_args.fd, 0, 0) == -1) {
+                    serverLog(LL_NOTICE,"Unable to reclaim page cache: %s", strerror(errno));
```

Comment:
maybe considering my other advise of using it on all OSes, not just linux, we wanna silently ignore the error?

Reply:
The reason to avoid logging here is to refrain from spamming the log with redundant messages, which could happen regardless of OS. We could add a counter instead, but given that it's very unlikely to hit an error here, a counter is probably not worth it.

Reply:
Normally reclaimFilePageCache won't throw a error, but I worry some odd kernel or fs may hit an error, and personally in my practice I prefer more observability in case some odd bug occurs. As the code here is only used on some file load operation, which is not much high frequent, I suppose it wouldn't produce many noise.

Reply:
ok, considering it's only when closing a file (not a high rate operation), i guess we can keep it.